docs/CRSF_SENSORS.md

```diff
@@ -0,0 +1,40 @@
+# mLRS Documentation: CRSF Sensors #
+
+([back to main page](../README.md))
+
+([back to CRSF page](CRSF.md))
+
+mLRS provides the following telemetry sensors when using CRSF:
+
+Num | Datapoint | Description                                                              | Data Source |
+--- | --------- | ------------------------------------------------------------------------ | ----------- |
+1   | 1RSS       | Receiver RSSI                                                           | mLRS        |
+2   | 2RSS       | Not supported                                                           |             |
+3   | Alt        | Altitude                                                                | FC          |
+4   | ANT        | Not supported                                                           |             |
+5   | Bat%       | Battery percent remaining                                               | FC          |
+6   | Capa       | Current consumption                                                     | FC          |
+7   | Curr       | Current draw                                                            | FC          |
+8   | FM         | Flight mode                                                             | FC          |
+9   | GPS        | GPS coordinates                                                         | FC          |
+10  | GSpd       | Ground speed                                                            | FC          |
+11  | Hdg        | Heading                                                                 | FC          |
+12  | Ptch       | FC pitch angle                                                          | FC          |
+13  | RFMD       | Receiver update rate (19, 31, 50)                                       | mLRS        |
```

Comment:
the values actually are 19, 31, 1 !!! This is because in CRSF language value 1 is for 50 Hz ... !!!
I'm not sure from top of my head if OTX/ETX display 1 as 50 Hz, pl check what is actually displayed

Reply:
You're right - 50Hz shows up as 1.  Added this to the description.